src/main/java/maksym/fedorenko/bookstore/model/Status.java

```diff
@@ -0,0 +1,11 @@
+package maksym.fedorenko.bookstore.model;
+
+public enum Status {
```

Comment:
it may be better to name it `OrderStatus`

also you can make it nested in `Order`